src/sweeps/config/schema.json

```
@@ -286,6 +286,30 @@
       },
       "additionalProperties": false
     },
+    "param_inv_loguniform": {
+      "type": "object",
+      "description": "inverse log uniform distribution",
+      "required": [
+        "distribution",
+        "max",
+        "min"
+      ],
+      "properties": {
+        "min": {
+          "description": "float",
+          "type": "number"
+        },
+        "max": {
+          "description": "float",
+          "type": "number"
+        },
+        "distribution": {
+          "enum": [
+            "inv_log_uniform"
+          ]
+        }
+      }
```

```suggestion
      },
      "additionalProperties": false
```
to make sure people dont add any additional fields to the parameter specs